src/Code/Enemies/Enemy.gd

```diff
@@ -59,3 +64,13 @@ func _get_dir_to_target() -> Vector2:
 		if t.length() < 10: return Vector2()
 		return t.normalized()
 	return Vector2()
+
+
+func _on_Hitbox_invincibility_started():
+	print("StartInvincibility")
```

Comment:
Remove extra print statements. You can use push_error or push-warning if this is something that should be tracked.